File: PRAK404-2210817120009-Najah_Maisyaroh.py
loop = 1
while loop == 1:
    print("Pilih program\n 1. Penjumlahan\n 2. Pengurangan\n 3. Perkalian\n 4. Pembagian\n 5. Exit")
    menu = int(input("Masukkan pilihan : "))
    if menu == 1: 
        pilihan = "penjumlahan"
        bil1=float(input ("Masukkan nilai pertama : "))
        bil2=float(input ("Masukkan nilai kedua   : "))
        hasil = bil1 + bil2
        print("Hasil",pilihan, "antara %0.2f dengan %0.2f adalah %0.2f \n" %(bil1, bil2, hasil))
    elif menu == 2: 
        pilihan = "pengurangan" 
        bil1=float(input ("Masukkan nilai pertama : "))
        bil2=float(input ("Masukkan nilai kedua   : "))
        hasil = bil1 - bil2
        print("Hasil",pilihan, "antara %0.2f dengan %0.2f adalah %0.2f \n" %(bil1, bil2, hasil))
    elif menu == 3: 
        pilihan = "perkalian"
        bil1=float(input ("Masukkan nilai pertama : "))
        bil2=float(input ("Masukkan nilai kedua   : "))
        hasil = bil1 * bil2
        print("Hasil",pilihan, "antara %0.2f dengan %0.2f adalah %0.2f \n" %(bil1, bil2, hasil))
    elif menu == 4:
        pilihan = "pembagian"
        bil1=float(input ("Masukkan nilai pertama : "))
        bil2=float(input ("Masukkan nilai kedua   : "))
        hasil = bil1 / bil2
        print("Hasil",pilihan, "antara %0.2f dengan %0.2f adalah %0.2f \n" %(bil1, bil2, hasil))
    elif menu == 5:
        nama = "Najah Maisyaroh"
        print(f"Terimakasih, telah menggunakan kalkulator {nama}")
        loop = 0
        break
    else:
        print("Input anda salah, silahkan coba lagi \n")
    continue

File: PRAK404-2210817120009-Najah_Maisyaroh.c
#include <stdio.h>
int main(){
    int menu;
    float bil1, bil2, hasil;
    char nama[]= "Najah Maisyaroh";
    while (menu!=5){
        printf("Pilih Program \n 1. Penjumlahan\n 2. Pengurangan\n 3. Perkalian\n 4. Pembagian\n 5. Exit\n");
            printf("Masukkan Pilihan : ");
            scanf("%d", &menu);
        if (menu > 5 || menu <= 0){
            printf("Input anda salah, silahkan coba lagi \n");}
        else if (menu == 5){
            printf("Terimakasih, telah menggunakan kalkulator %s", nama);}
        else{
            printf("Masukkan nilai pertama : ");
            scanf("%f", &bil1);
            printf("Masukkan nilai kedua   : ");
            scanf("%f",  &bil2);
            if(menu==1){
                hasil = bil1 + bil2;
                printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", bil1, bil2, hasil);}
            else if(menu==2){
                hasil = bil1 - bil2;
                printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", bil1, bil2, hasil);}
            else if(menu==3){
                hasil = bil1 * bil2;
                printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n", bil1, bil2, hasil);}
            else{
                hasil = bil1 / bil2;
                printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n", bil1, bil2, hasil);}}}
}